addons/binding/org.openhab.binding.rme/ESH-INF/thing/rme.xml

```diff
@@ -0,0 +1,95 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="rme"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<!-- Sample Thing Type -->
+	<thing-type id="manager">
+		<label>RME Rain Manager</label>
+		<description>The RME Rain Manager is a controllable water supply unit that can monitor a gauge in a cistern, and switch between rain water and city provided water</description>
+
+		<channels>
+			<channel id="waterlevel" typeId="waterlevel" />
+			<channel id="mode" typeId="mode" />
+			<channel id="source" typeId="source" />
+			<channel id="exitpump" typeId="statusflag">
+			     <label>Exit Pump Active</label>
+			</channel>
+			<channel id="entrypump" typeId="statusflag">
+			     <label>Entry Pump Active</label>
+			</channel>
+			<channel id="waterexchange" typeId="statusflag">
+			     <label>Water Exchange Active</label>
+			</channel>
+			<channel id="cisternsupplyalarm" typeId="statusflag">
+			     <label>Cistern Supply Alarm</label>
+			</channel>
+			<channel id="overflowalarm" typeId="statusflag">
+			     <label>Overflow Alarm</label>
+			</channel>
+			<channel id="cisternblockedalarm" typeId="statusflag">
+			      <label>Cistern Blocked Alarm</label>
+			</channel>
+			<channel id="filtercleaning" typeId="statusflag">
+			       <label>Filter Cleaning Required</label>
+			</channel>
+		</channels>
+
+		<config-description>
+			<parameter name="port" type="text">
+				<label>Serial Port</label>
+				<description>Serial Port the RME Rain Manager is connected to</description>
+				<required>true</required>
+			</parameter>
+		</config-description>
+	</thing-type>
+
+	<!-- Water Level Channel Type -->
+	<channel-type id="waterlevel">
+		<item-type>Number</item-type>
+		<label>Water Level</label>
+		<description>Indicates the % the cistern is filled with water</description>
+		<state readOnly="true" pattern="%.1f %%" />
+	</channel-type>
+
+	<!-- Operation Mode Channel Type -->
+	<channel-type id="mode">
+		<item-type>String</item-type>
+		<label>Operation Mode</label>
+		<description>Indicates the operation mode of the RME Rain Manager</description>
+		<state readOnly="true">
+	       <options>
+	           <option value="Automatic">Automatic</option>
+	            <option value="Manual">Manual</option>
+	       </options>
+        </state>
+	</channel-type>
+	
+		<!-- Operation Mode Channel Type -->
+	<channel-type id="source">
+		<item-type>String</item-type>
+		<label>Water Source</label>
+		<description>Indicates water source used to supply water, e.g cistern or city</description>
+        <state readOnly="true">
+           <options>
+               <option value="Rain">Rain</option>
+                <option value="City">City</option>
```

Comment:
there is inconsistent identation - could you please apply the code formatter on all XML files?